lua/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Copy and adapt the `build.sh` script from [Nginx](https://github.com/unikraft/static-pie-apps/tree/master/nginx). To have a similar structure, comments etc.